packages/wrangler/src/config/config.ts

```diff
@@ -30,6 +30,8 @@ export type RawConfig = Partial<ConfigFields<RawDevConfig>> &
 export interface ConfigFields<Dev extends RawDevConfig> {
 	configPath: string | undefined;
 
+	metadata_binding: string | undefined;
```

Comment:
I believe this should be part of the environment configuration, just like our other binding types:
https://github.com/cloudflare/wrangler2/blob/02ca556c3e84d45cb3eaa5787a4a0ed5254c3815/packages/wrangler/src/config/environment.ts#L259